### PR TITLE
Remarks

### PR DESCRIPTION
**Pull request by miguelsierte**
Status and remarks for admin
Screenshot 2023-08-30 225312

### CODE REVIEW

**Comment by jrglomar** [comment]
> Status and remarks for admin Screenshot 2023-08-30 225312

Please, provide more screenshot of 

I. Where remarks displayed
II. Other status value

File: app/Http/Controllers/ArtworkController.php [hunk 1]
```diff
@@ -42,7 +42,8 @@ public function store(Request $request)
         'end_date' => $validatedData['end_date'],
         'users_id' => Auth::id(),
         'category_id' => $validatedData['category_id'],
-        'status' => false,
+        'status' => 'pending',
```
Comment (jrglomar) [inline app/Http/Controllers/ArtworkController.php:45]: Can you check if you can remove this. I believe you already set a default status value on database migration config.

File: app/Http/Controllers/ArtworkController.php [hunk 2]
```diff
@@ -80,7 +81,9 @@ public function forsalePost(Request $request)
         'dimension' => $validatedData['dimension'],
         'price' => $validatedData['price'],
         'users_id' => Auth::id(),
-        'category_id' => $validatedData['category_id']
+        'category_id' => $validatedData['category_id'],
+        'status' => 'pending', // Default status
```
Comment (jrglomar) [inline app/Http/Controllers/ArtworkController.php:85]: Same here. Can you check if you can remove this. I believe you already set a default status value on database migration config.

**Comment by miguelsierte** [comment]
Screenshot 2023-08-31 013859

**Comment by miguelsierte** [comment]
Screenshot 2023-08-31 014232
Screenshot 2023-08-31 014153
Screenshot 2023-08-31 013859

**Review by jrglomar — APPROVED** [review]

**Pull request merged**